src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -126,6 +125,11 @@
 
 	private Clock clock = Clock.systemDefaultZone();
 
+	@Override
+	public Log getLog() {
+		return new StripSensitiveDataLog(super.getLog());
+	}
```

Comment:
maybe store result and reuse ... now every call to get Log will create new class
I didn't analize how many times getLog is called

Reply:
I think it is questionable if this should be cached. There is no definition if `getLog` can be cached. The method is called max 4-5 times for an Mojo execution and i personally think this is ok not to be cached.

However i've added a cache in #167.

It would be better to avoid logging credentials in code and not inside a log delegate. But this would need a more complex refactoring.